src/client/Contracts.js

```diff
@@ -572,3 +574,113 @@ exports.SendFunds = async function({recipient, ether}) {
 
   return await transaction.wait();
 };
+
+/**
+ * TenantContractId returns the ID of the tenant contract for
+ * the objectId provided from contract metadata or from fabric metadata
+ *
+ * @param objectAddress
+ * @returns {Promise<string|undefined>}
+ */
+exports.TenantContractId = async function(objectAddress) {
+  ValidateAddress(objectAddress);
+
+  const objectId = Utils.AddressToObjectId(objectAddress);
+
+  const hasGetMetaMethod = await this.authClient.ContractHasMethod({
+    contractAddress: objectAddress,
+    methodName: "getMeta"
+  });
+
+  if(hasGetMetaMethod) {
+    const tenantContractId = await this.ContractMetadata({
+      contractAddress:objectAddress,
+      metadataKey:"_ELV_TENANT_ID"
+    });
+    if(tenantContractId !== "") {
+      return tenantContractId;
+    }
+  }
+
+  const libraryId = await this.ContentObjectLibraryId({
+    objectId: Utils.AddressToObjectId(objectAddress)
+  });
+
+  return await this.ContentObjectMetadata({
+    libraryId,
+    objectId,
+    metadataSubtree: "tenantContractId",
+  });
+};
+
+/**
+ * SetTenantContractId sets the ID of the tenant contract for the
+ * objectAddress provided in contract metadata or in fabric metadata
+ *
+ * @param objectAddress
+ * @param tenantContractId
+ * @returns {Promise<{tenantId: (undefined|string), tenantContractId: string}>}
+ */
+exports.SetTenantContractId = async function(objectAddress, tenantContractId) {
```

Comment:
Same here, named params, contractAddress instead of objectAddress, add objectId and versionHash